recaf-launcher/src/main/java/dev/xdark/recaf/Launcher.java

```diff
@@ -319,16 +311,15 @@ private static String getLocalVersion(Path jarPath) {
 	}
 
 	/**
-	 * @return {@code true} when {@link #baseDir} successfully set.
+	 * Sets Recaf's base directory.
 	 */
-	private static boolean setBaseDir() {
+	private static void setBaseDir() {
 		try {
 			baseDir = PathUtils.getRecafDirectory();
 			logger.info("Recaf directory: {}", baseDir);
-			return true;
 		} catch (IllegalStateException ex) {
 			logger.error("Unable to get base directory: ", ex);
```

Comment:
`: ` seems to indicate something should be logged. Should this be a `.`?

Reply:
Slf4j will log exception afterwards